airbyte-cdk/java/airbyte-cdk/core/src/test/kotlin/io/airbyte/cdk/integrations/destination/async/AsyncStreamConsumerTest.kt

```diff
@@ -146,9 +145,9 @@ class AsyncStreamConsumerTest {
                 onClose = onClose,
                 onFlush = flushFunction,
                 catalog = CATALOG,
-                bufferManager = BufferManager(),
+                bufferManager = BufferManager("default_ns"),
```

Comment:
I feel like there's still something funky here. We need to pass the default namespace twice. Seems to be an improvement though

Reply:
yep. I really wish we had some sort of wrapper/replacement object for RecordMessage, so we could track `real_namespace: String?` alongside `nonnull_namespace: String`, and not need to worry about "this half of our codebase operates on nullable namespace and the other half operates on nonnullable"

b/c then we could just set the nonnull namespace immediately after reading the message off of stdin, and it's up to each component whether they want to use the real namespace or the nonnull version

Reply:
can we reference the namespace from the bufferManager object? That would be slightly cleaner

Reply:
hm, not sure what you mean by that - buffer manager doesn't have a reference to anything that knows the default namespace, and I don't think we want to give it a reference to AsyncStreamConsumer?

(I _should_ switch this to use the DEFAULT_NAMESPACE constant though)